### PR TITLE
Add workspace endpoint

### PR DESCRIPTION
**Pull request by lgarvey**
Add a separate API endpoint for digital workspace to ingest all company data

The endpoint ... 
supplies the company source data fields: source and worldbase_source
is paginated
is protected by token authentication

### CODE REVIEW

File: workspace/models.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from django.db import models
```
Comment (currycoder) [inline workspace/models.py:1]: This file could be deleted

Comment (lgarvey) [reply workspace/models.py:1]: AFAIK the models.py file and __init__.py files are needed, even if empty

Comment (currycoder) [reply workspace/models.py:1]: I believe models.py is optional.  You're absolutely right about `__init__.py` though

Comment (lgarvey) [reply workspace/models.py:1]: Ah cool - looks like pre django 1.7 they were required.  I'll remove it.

**Pull request merged**